packages/facade/src/apis/sheets/f-worksheet.ts

```diff
@@ -31,6 +31,10 @@ export class FWorksheet {
         @ICommandService private readonly _commandService: ICommandService
     ) {}
 
+    getId(): string {
```

Comment:
In order to reduce the learning cost for facade API users, it is expected to align with the Google Apps Script API, https://developers.google.com/apps-script/reference/spreadsheet/sheet#getSheetId()

Reply:
resolved on #1770